pkg/util/util.go

```diff
@@ -220,6 +222,21 @@ func KustomizeHash(obj client.Object) (hash string, err error) {
 	return kustomizeHasher.Hash(objKYaml)
 }
 
+// GenerateHashFromStrings generates a hash from a list of strings. This hash can be used to identify resources and
+// if they can be removed. E.g. the ogcapi-operator provides the volume-operator with a hash so it knows what resources
+// to create or remove.
+func GenerateHashFromStrings(sliceToHash []string) string {
+	// Concatenate all strings to hash
+	var data string
+
+	for _, val := range sliceToHash {
+		data += val
+	}
+
+	hash := sha256.Sum256([]byte(data))
+	return hex.EncodeToString(hash[:8])
```

Comment:
Is er een reden om dit in te korten? Zo ja misschien even als comment opnemen.

Reply:
Sha256 heeft een vaste lengte. Maar als het echt kort moet zijn dan kan dit een oplossing zijn. Er is een minimale kans op collisions.